src/psyclone/domain/lfric/kernel/field_arg.py

```diff
@@ -74,10 +78,18 @@ def create_from_fparser2(fparser2_tree):
         :returns: an instance of FieldArg.
         :rtype: :py:class:`psyclone.domain.lfric.kernel.FieldArg`
 
+        raises ValueError: if the metadata is not in the correct form.
+
         '''
         FieldArg.check_fparser2(fparser2_tree, nargs=4)
```

Comment:
I think you can use `self` rather than FieldArg, even though it's a class method.

Reply:
This is within a static method so there is no 'self'.

Reply:
Oh yes, I was just testing ;-)